examples/lib/defaultHtmlStepUi.js

```diff
@@ -127,12 +135,12 @@ function DefaultHtmlStepUi(_sequencer, options) {
           html +
           '\
                          </div>';
-        step.ui.querySelector('div.details').appendChild(div);
+        $step('div.details').append(div);
```

Comment:
So now we will do append and not appendChild as step is now a jquery element. Just for my knowledge

Reply:
Yes, it is fully jquery now. Just think of it as `$` not `$step`

Reply:
yeah great!